modules/proc_installing-the-rollout-manager-plugin.adoc

```diff
@@ -0,0 +1,106 @@
+:_mod-docs-content-type: PROCEDURE
+[id="installing-the-rollout-manager-plugin_{context}"]
+= Installing the rollout-manager plugin
+
+Once the rollout controller is created, you must install the rollout-manager plugin.
+
+.Prerequisites
+
+* The `cluster-admin` cluster role must be assigned to the user.
+
+.Procedure
+
+. Run the `yaml/rbac.yaml` file to add the role for running the required operations on the Openshift Route and build this plugin.
+
+. Save this plugin in your local system and mount it on the argo-rollouts container:
+
+.. Apply the rbac permissions:
++
+[source,terminal]
+----
+kubectl apply -f https://raw.githubusercontent.com/argoproj-labs/rollouts-plugin-trafficrouter-openshift/main/yaml/rbac.yaml
+----
+
+.. Create an instance of argo-rollouts by creating a file with the following configuration:
++
+[source,terminal]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: argo-rollouts
+  namespace: argo-rollouts
+spec:
+  template:
+    spec:
+      ...
+      volumes:
+        ...
+         - name: openshift-route-plugin
+           hostPath:
+             path: /CHANGE-ME/rollouts-plugin-trafficrouter-openshift
+             type: ''
+      containers:
+        - name: argo-rollouts
+        ...
+          volumeMounts:
+             - name: openshift-route-plugin
+               mountPath: /CHANGE-ME/rollouts-plugin-trafficrouter-openshift
+----
+
+.. Add the following configuration to a YAML file:
++
+[source,terminal]
+----
+kubectl apply -f <path_to_file>
+----
+
+.. Create a ConfigMap so that the `argo-rollouts` instance knows the location of the plugin.
```

Comment:
This step is now optional. User has the privilege to configure a customization plugin or use this option.

Reply:
Fixed.